### PR TITLE
saves the bookmark lists if the app is discard directly or indirectly…

### PR DESCRIPTION
**Pull request by pravalgautam**
…  Closes-  #4740

### CODE REVIEW

File: iphone/Maps/Bookmarks/Categories/BMCView/BMCViewController.swift [hunk 1]
```diff
@@ -47,7 +47,7 @@ final class BMCViewController: MWMViewController {
     // Disable all notifications in BM on appearance of this view.
     // It allows to significantly improve performance in case of bookmarks
     // modification. All notifications will be sent on controller's disappearance.
-    viewModel.setNotificationsEnabled(false)
+    viewModel.setNotificationsEnabled(true)
```
Comment (biodranik) [inline iphone/Maps/Bookmarks/Categories/BMCView/BMCViewController.swift:50]: Please read the comment. There was a reason to disable notifications. Can it be fixed by enabling only the one necessary notification?

Comment (pravalgautam) [reply iphone/Maps/Bookmarks/Categories/BMCView/BMCViewController.swift:50]: I tested it and it works

Comment (biodranik) [reply iphone/Maps/Bookmarks/Categories/BMCView/BMCViewController.swift:50]: Did you test it with hundreds of bookmark lists with thousands of bookmarks?

Comment (rtsisyk) [reply iphone/Maps/Bookmarks/Categories/BMCView/BMCViewController.swift:50]: How does it fix #4740? I am not following...

**Pull request closed without merging**